### PR TITLE
[Feature] Start Learning Block

### PR DESCRIPTION
**Pull request by KobbyMmo**
## Description

Add start Learning Block as described by Tast

Fixes # ([179117185](https://www.pivotaltracker.com/story/show/179117185))

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👍🏾 

------
1. Background missingScreenshot from 2021-08-12 12-59-20
2. What is supposed to happen when **Learn more** is clicked?
3. CMS?

File: src/components/StartLearning/index.stories.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+import StartLearning from ".";
+
+export default {
+  title: "Components/Start Learning",
```
Comment (kilemensi) [inline src/components/StartLearning/index.stories.js:6]:
```suggestion
  title: "Sections/StartLearning",
```

File: src/components/StartLearning/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  },
+  root: {
+    backgroundImage: `url('${mapLines}')`,
+    backgroundSize: "cover",
+    backgroundRepeat: "no-repeat",
+    paddingTop: typography.pxToRem(80),
+    paddingBottom: typography.pxToRem(80),
+  },
+  subtitle: {
+    fontWeight: 500,
```
Comment (KhadijaMahanga) [inline src/components/StartLearning/index.js:22]:
```suggestion
    fontWeight: 600,
```

File: src/components/StartLearning/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    marginTop: typography.pxToRem(40),
+    marginBottom: typography.pxToRem(40),
+  },
```
Comment (KhadijaMahanga) [inline src/components/StartLearning/index.js:25]: the tagline or subtitle should have `#212529` color

**Review by KhadijaMahanga — COMMENTED** [review]
yeah..background image missing

**Review by nerdlet — APPROVED** [review]

File: src/components/StartLearning/index.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+import PropTypes from "prop-types";
+import React from "react";
+
+import mapLines from "@/pesayetu/assets/images/Mask Group 8@2x.png";
+import Header from "@/pesayetu/components/Header";
+import Section from "@/pesayetu/components/Section";
+
+const useStyles = makeStyles(({ typography }) => ({
+  root: {
+    backgroundImage: `url('${mapLines}')`,
```
Comment (kilemensi) [inline src/components/StartLearning/index.js:12]: Isn't `mapLines` going to be an object?

File: src/components/StartLearning/index.stories.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+import StartLearning from ".";
+
+export default {
+  title: "Components/StartLearning",
```
Comment (kilemensi) [inline src/components/StartLearning/index.stories.js:6]:
```suggestion
  title: "Sections/StartLearning",
```

File: src/components/StartLearning/index.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    backgroundRepeat: "no-repeat",
+    paddingTop: typography.pxToRem(80),
+    paddingBottom: typography.pxToRem(80),
+  },
+  section: {
+    textAlign: "center",
+  },
+  subtitle: {
+    fontWeight: 600,
+    color: "#212529",
```
Comment (kilemensi) [inline src/components/StartLearning/index.js:23]:
```suggestion
```
Where are my [performance review papers](https://github.com/CodeForAfrica/PesaYetu/blob/7dfb5440eb583046b976326910bcc0b248e17c73/src/theme/index.js#L135-L140)?

**Review by kilemensi — APPROVED** [review]
🚀

File: src/components/StartLearning/index.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  return (
+    <div className={classes.root}>
+      <Section classes={{ root: classes.section }}>
+        <Header>{title}</Header>
+        <Typography variant="h3" className={classes.subtitle}>
+          {subtitle}
+        </Typography>
+        <Button href={href} variant="text">
+          {ctaText}
+        </Button>
```
Comment (KhadijaMahanga) [inline src/components/StartLearning/index.js:38]:
```suggestion
        <Button 
        component={Link}
        href={href} variant="text" >
          {ctaText}
        </Button>
```

**Review by KhadijaMahanga — APPROVED** [review]
🚀

**Pull request merged**